PracticeROM/homeboy
# Build D43J version (mq-j) and boot gz

GC gz now boots at least. The main changes compared to Wii are removing reset and SD card functionality, and reducing the ROM cache size, and making the gz loader DMA work.

To test:
* Build and install gzinject from https://github.com/PracticeROM/gzinject
* Check out https://github.com/cadmic/gz/tree/gc-build, update homeboy submodule, and run `./make-iso <mq-j ISO>`
